tools/db_stress.cc

```diff
@@ -2372,6 +2372,11 @@ class StressTest {
       }
 #endif
     }
+    while (!thread->snapshot_queue.empty()) {
+      db_->ReleaseSnapshot(thread->snapshot_queue.front().second.snapshot);
```

Comment:
I wonder why this problem did not show up before. I am not sure why it should be triggered by the recent changes.